app/frontend/src/AppRoutes.tsx

```diff
@@ -3,7 +3,7 @@ import { Switch, Route, RouteProps, Redirect } from "react-router-dom";
 import Login from "./features/auth/login/Login";
 import ProfilePage from "./features/profile/ProfilePage";
 import Home from "./features/Home";
-import Messages from "./features/Messages";
+import Messages from "./features/messages/index";
```

Comment:
Why this change?

Reply:
Messages is a file, messages is a directory.